codegen-winter/src/air/transition_constraints.rs

```diff
@@ -83,8 +83,17 @@ impl Codegen for Operation {
 
                 strings.push(rhs);
             }
+            Operation::Exp(l_idx, r_idx) => {
+                let lhs = l_idx.to_string(graph);
+                strings.push(lhs);
+
+                strings.push(String::from("^"));
+
+                let rhs = r_idx.to_string();
+                strings.push(rhs);
+            }
```

Comment:
I have the same thought here that I left in the `mul` PR - I think using `format!` would be better.